src/less/less/variables-common.less

```diff
@@ -6,4 +6,5 @@
 @marketsans-filename-regular: "MarketSans-Regular-WebS";
 @marketsans-filename-bold: "MarketSans-SemiBold-WebS";
 
+@font-size-12: 0.75rem;
```

Comment:
Does this make it 10.5px (e.g. 11px) for DS4, since DS4's base is 14px? Seems pretty small.